spec/features/user_deletes_a_spot_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature "User deletes a spot" do
+
+  # As a user I want to be able to edit a spot I created
+  # so that the information is up-to-date.
+  # Acceptance criteria:
+  # User can edit the spot they created.
+
+  scenario 'I want to delete a study spot' do
+    skip
```

Comment:
It looks like you have a `destroy` action in your spots controller - you should either fully implement and test that feature, or not include it all! It's a bad idea to merge half-finished work into master.